libexec/bats

```diff
@@ -128,15 +132,14 @@ if [[ "${#options[@]}" -ne '0' ]]; then
       pretty="1"
       ;;
     * )
-      usage "Bad command line option '-$option'" >&2
-      exit 1
+      abort "Bad command line option '-$option'"
       ;;
     esac
   done
 fi
 
 if [ "${#arguments[@]}" -eq 0 ]; then
-  usage "Must specify at least one <test>" >&2
+  abort 'Must specify at least one <test>'
   exit 1
```

Comment:
This 'exit 1' on line 143 is superfluous with the introduction of the `abort()` function.

Reply:
Whoops...I seemed to miss that in the original commit, but fixed it in the merge.